ecommerce/models.py

```diff
@@ -80,6 +80,9 @@ class Position(models.Model):
     name = models.CharField(max_length=255)
     quantity = models.PositiveSmallIntegerField(default=0)
     price = models.FloatField(blank=True, null=True)
+    code = models.CharField(max_length=255, default='')
+    catalog_name = models.CharField(max_length=2047, default='')
```

Comment:
`2047` looks cryptic. Let's comment it to justify the value

Also probably it would be better to use `TextField` instead of `CharField`

Reply:
@artemiy312 , 255 is `2**8 - 1`, so i just provided `2**11 - 1`.
Seems it's not good to use TextField for name field. I'll provide value as `2**10 - 1`. Seems it should be enough